.github/workflows/test-suite-web-e2e.yaml

```diff
@@ -0,0 +1,84 @@
+name: "[Test] e2e suite-web"
```

Comment:
.yml or .yaml? this is the first and only .yaml

Reply:
yeah, well yaml is the correct one, yml was historically used because of Windows as it did not support longer that 3 characters file extensions :D we can keep using yml its just that I've lately using yaml everywhere, so that's why I put it here also just a habit